### ISSUE TITLE
Libre office writer support

### ISSUE THREAD

**Issue by sp55000** [nvaccess/nvda#6284]
Please ad support for LibreOffice in NVDA. Because in India, many blind people cannot purchase MS Office by paying thousands of rupees. NVDA being a free screen reader, is certainly a boon here. Thank you.

**Comment by josephsl** [comment]
Hi, NVDA provides basic support for LibreOffice at this time. Can you clarify what’s needed? Thanks.

**Comment by javidominguez** [comment]
Basic support provided by NVDA works reasonably well for simple documents, but does not report the alignment of the paragraph. I'm using Libre Office Writter 5.1

**Comment by Brian1Gaff** [comment]
There are quite a lot of issues with both Libra and Open Office Still the 
say all does not work reliably and as has been said one keeps forgetting its 
not word and wondering why some stuff is not working.
 I do think the constantly changing code of the programs might have 
something to do with this. I have now reinstalled an old copy of Office XP 
and given up on the free office as its too quirky.
Although nvda does not officially support his old version of word, it still 
works better in nvda than Libra does.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by feerrenrut** [comment]
@supriyasuppy could you please comment on the particular feature/behaviour that NVDA is missing when used with Libre office? I understand that there may be many features missing or broken, however in order for us to know where to focus our efforts we require more specific bug reports or feature requests. If there are multiple issues, please create a Github issue for each.

An example of this could be: 
> When using LibreOffice Writer 5.1 I set the paragraph indentation by going to the "Format" menu, selecting the "paragraph" option, then on the indents and spacing tab I set the "before text" value to 2cm. In NVDA I have the "paragraph indentation" option enabled (via the document formatting dialog). When I use LibreOffice Writer with NVDA and move by line between paragraphs that have different indentation settings the indentation level is not reported. I expect it to say something like 'no left indent' and 'left indent 2 centimeters'.

**Comment by supriyasuppy** [comment]
Hello,
I think these issues need attention in LibreOffice Writer:
1. Table navigation commands do not work when pressed inside a table - Eg. Alt plus Control plus Left/right Arrow keys have no effect.
2. When reading by paragraph, sometimes NVDA becomes silent when pressed Control plus Down arrow - I have to press again Control plus Down arrow and Control plus Up arrow in order to read the skipped paragraph.
Thanks for your assistance. Please keep up the great work.

**Comment by bhavyashah** [comment]
@sp55000 As has been suggested earlier, could you please file a separate Github ticket for each LibreOffice accessibility bug individually? Additionally, sending feedback to LibreOffice folks may also help in establishing a bridge between devs of LO and NVDA. Since this particular ticket has nothing specific, can we safely close it? @michaelDCurran @leonardder @Qchristensen

**Comment by feerrenrut** [comment]
Closing this issue, since it is not describing one particular issue. We are happy to accept and look into the problems that have been mentioned here if they are created and described in individual issues. Please consider adding an example document when creating these issues. This will make it much easier for the person who is to try to fix these issues. Thanks.
